tests/functional/object/mcg/test_bucket_delete_using_obc_creds.py

```diff
@@ -0,0 +1,82 @@
+import base64
+import boto3
+import logging
+
+from ocs_ci.framework.pytest_customization.marks import (
+    tier2,
+    bugzilla,
+    red_squad,
+    runs_on_provider,
+    skipif_ocs_version,
+    polarion_id,
+    mcg,
+)
+from ocs_ci.ocs.resources.bucket_policy import HttpResponseParser
+from ocs_ci.ocs.ocp import OCP
+import botocore.exceptions as boto3exception
+from ocs_ci.ocs.constants import (
+    OPENSHIFT_STORAGE_NAMESPACE,
+    SECRET,
+)
+from ocs_ci.ocs.exceptions import UnexpectedBehaviour
+
+logger = logging.getLogger(__name__)
+
+
+@tier2
+@mcg
+@red_squad
+@runs_on_provider
+@skipif_ocs_version("<4.17")
+@polarion_id("OCS-6252")
+@bugzilla("2301657")
+def test_bucket_delete_using_obc_creds(mcg_obj, bucket_factory):
+    """
+    Test bucket deletion using OBC generated credentials and noobaa admin credentials
+
```

Comment:
Please add the expected behavior in the Docstring, especially since it's a new expectation that was just introduced in the fix of the covered BZ:
```suggestion
    Verify that deletion of an OBC's bucket is denied when using the
    OBC's credentials, but permitted with the noobaa admin credentials.    

```

Reply:
Fixed